broadcast/internal/arc/arc_submit_tx.go

```diff
@@ -189,33 +189,22 @@ func appendSubmitTxHeaders(pld *httpclient.HTTPRequest, opts *broadcast.Transact
 }
 
 func decodeSubmitResponseBody(resp *http.Response) (*broadcast.SubmittedTx, error) {
-	base := broadcast.BaseSubmitTxResponse{}
-	err := arc_utils.DecodeResponseBody(resp.Body, &base)
+	model := &broadcast.SubmittedTx{}
+	err := arc_utils.DecodeResponseBody(resp.Body, &model)
```

Comment:
I'm not sure if passing pointer to pointer is a good idea here.

Reply:
Fixed